docs/get-started/getting-started-alluxio.md

```diff
@@ -365,3 +365,77 @@ This section will give some links about how to configure, tune Alluxio and some
 - [Alluxio troubleshooting](https://docs.alluxio.io/os/user/stable/en/operation/Troubleshooting.html)
+
+## Alluxio reliability
+The properties mentioned in this section can be found in [Alluxio configuration](https://docs.alluxio.io/os/user/stable/en/reference/Properties-List.html)
+### Auto start Alluxio masters and workers
+After installed Alluxio masters and workers, use this [script](../../scripts/alluxio/alluxio-systemd.sh) to create systemd services and start masters and workers.
+#### Prerequisites
+- Format Alluxio on masters: 
+    ```bash
+    $ ./bin/alluxio format
+    ```
+    Or you have executed `alluxio-start.sh master` which internally invoke `alluxio format`  
+- Close all the Alluxio masters and workers
+#### Create systemd services
+Commands are:
+```bash
+# create systemd service for master and start master on master node
+sudo sh -c 'ALLUXIO_HOME=/opt/alluxio-2.8.0 RUN_USER=ubuntu ./alluxio-systemd.sh master'
+
+# create systemd service for worker and start worker on worker node
+sudo sh -c 'ALLUXIO_HOME=/opt/alluxio-2.8.0 RUN_USER=ubuntu ./alluxio-systemd.sh worker'
+```
+Please specify ALLUXIO_HOME and RUN_USER properties if you have different properties.
+
+Now, the masters and workers can automatically start if the host rebooted or the processes of masters and workers are unexpected killed.
+If you want to stop the services, please do not use `kill`, use:
+```bash
+sudo systemctl stop alluxio-master.service
+sudo systemctl stop alluxio-worker.service
+```
+If you want to restart the services, please use:
+```bash
+sudo systemctl restart alluxio-master.service
+sudo systemctl restart alluxio-worker.service
+```
+
+### Avoid long time response if masters or workers have failures
+The default value of `alluxio.user.rpc.retry.max.duration` is 2 minutes.  
+If master is exited, the `alluxio.user.rpc.retry.max.duration` will cause at least 2 minutes waiting, it's too long.
+The default value of `alluxio.user.block.read.retry.max.duration` is 5 minutes.
+If worker is exited, the `alluxio.user.block.read.retry.max.duration` will cause 5 minutes waiting, it's too long.
+
+See also:
+```
+alluxio.user.rpc.retry.max.duration
+alluxio.user.rpc.retry.max.sleep	
+alluxio.user.rpc.retry.base.sleep	
+
+alluxio.user.block.read.retry.max.duration
+alluxio.user.block.read.retry.sleep.max		
+alluxio.user.block.read.retry.sleep.base 		
+```
+Above configurations define the `ExponentialTimeBoundedRetry` retry policies and `max durations`, we can adjust them to appropriate values.  
+
+Set these properties on Spark because Spark invokes Alluxio client.
+vi $SPARK_HOME/conf/spark-defaults.conf
+```
+spark.driver.extraJavaOptions    -Dalluxio.user.rpc.retry.max.duration=10sec -Dalluxio.user.block.read.retry.max.duration=10sec
+spark.executor.extraJavaOptions  -Dalluxio.user.rpc.retry.max.duration=10sec -Dalluxio.user.block.read.retry.max.duration=10sec
+```
+
+### Kick out dead worker early to avoid failures
```

Comment:
I think I would just call this Worker Timeout to fail fast.

Do we know the failure scenario's here?  would the worker be dead and thus systemd restarted or have we seen cases worker just hangs?   The reason I ask is there are failing fast and then intermittent issues... like say worker was Gc'ing and then we time it out.  I assume it can join back in?

Reply:
> Do we know the failure scenario's here?
I tested the killing process.

> would the worker be dead and thus systemd restarted or have we seen cases worker just hangs? 
Until now does not see other scenarios that worker hangs.
Long time GC obviously will cause the worker to hang. 

> The reason I ask is there are failing fast and then intermittent issues... like say worker was Gc'ing and then we time it out. I assume it can join back in?

Yes, If a worker fails fast and then comes back, it can join again.